Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Use the official Liquibase image as the base
+FROM liquibase/liquibase:4.29.1
```

Comment:
So will this version change via dependabot whenever a new tag is released from the main docker image?

Reply:
yes

Reply:
Got it. So there will be a slight delay in publishing the AWS container version. That makes sense.

Reply:
yes, as the approval from official docker team takes sometime.